recipes/r-enchantr/meta.yaml

```diff
@@ -42,30 +46,33 @@ requirements:
     - r-rmarkdown
     - r-scales
     - r-scoper >=1.2.1
-    - r-shazam >1.1.0
+    - bioconda::r-shazam >1.1.0
```

Comment:
Desperate, but... can we try with r-shazam >= 1.1.1? 1.1.0 has kedd as dependency, but this package has been removed from CRAN. I wonder if this could be causing trouble.